### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getScrollTop' of null

### ISSUE THREAD

**Issue by anjerodesu** [atom-minimap/minimap#380]
[Enter steps to reproduce below:]

1. Open 2 tabs and scroll the first tab down
2. Navigate to the second tab
3. Move cursor on mini-map
4. Close the other tab

**Atom Version**: 1.0.7
**System**: Mac OS X 10.10.5
**Thrown From**: [minimap](https://github.com/atom-minimap/minimap) package, v4.13.2

### Stack Trace

Uncaught TypeError: Cannot read property 'getScrollTop' of null

```
At /Users/zyyx-angelo/.atom/packages/minimap/lib/minimap.coffee:216

TypeError: Cannot read property 'getScrollTop' of null
  at Minimap.module.exports.Minimap.getTextEditorScrollRatio (/Users/zyyx-angelo/.atom/packages/minimap/lib/minimap.coffee:216:16)
  at Minimap.module.exports.Minimap.getCapedTextEditorScrollRatio (/Users/zyyx-angelo/.atom/packages/minimap/lib/minimap.coffee:223:50)
  at Minimap.module.exports.Minimap.getScrollTop (/Users/zyyx-angelo/.atom/packages/minimap/lib/minimap.coffee:334:17)
  at Minimap.module.exports.Minimap.getFirstVisibleScreenRow (/Users/zyyx-angelo/.atom/packages/minimap/lib/minimap.coffee:316:17)
  at Object.screenPositionForMinimapOffset (/Users/zyyx-angelo/.atom/packages/minimap-codeglance/lib/minimap-codeglance.js:142:46)
  at Object.scrollToMinimapOffset (/Users/zyyx-angelo/.atom/packages/minimap-codeglance/lib/minimap-codeglance.js:130:31)
  at atom-text-editor-minimap.<anonymous> (/Users/zyyx-angelo/.atom/packages/minimap-codeglance/lib/minimap-codeglance.js:89:21)

```

### Commands

```
 12x -4:25.9.0 application:new-file (atom-text-editor.editor.is-focused)
  3x -4:23 core:close (atom-text-editor.editor.is-focused)
     -4:21 core:undo (atom-text-editor.editor.is-focused)
  5x -4:19.8.0 pane:reopen-closed-item (atom-text-editor.editor.is-focused)
 39x -4:15.9.0 core:close (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "git-difftool",
      "git-plus",
      "package-sync",
      "ruby-test",
      "refactor",
      "wrap-guide",
      "jshint"
    ],
    "themes": [
      "one-dark-ui",
      "base16-tomorrow-dark-theme"
    ]
  },
  "minimap": {
    "plugins": {
      "git-diff": true,
      "codeglance": true
    }
  }
}
```

### Installed Packages

```coffee
# User
Stylus, v1.2.1
atom-beautify, v0.28.11
docblockr, v0.7.3
file-icons, v1.6.9
git-diff-details, v0.20.0
highlight-selected, v0.10.1
language-generic-config, v0.2.0
language-latex, v0.6.1
latex, v0.25.1
linter-javac, v1.1.0
minimap, v4.13.2
minimap-codeglance, v0.4.0
minimap-git-diff, v4.1.8
webbox-color, v0.5.8
wordcount, v2.3.0

# Dev
No dev packages
```

**Comment by abe33** [comment]
Hi @anjerodesu, thanks for taking the time to open a new issue. It appears the issue is raised from the codeglance plugin, I'll try to see with @olmokramer how to fix that.

**Comment by anjerodesu** [comment]
Sorry about that. I did not have the time to even check that it is from another plugin. Thanks as well for reporting to the proper authority. 😅

**Comment by olmokramer** [comment]
This should be fixed in minimap-codeglance v0.4.1 which I just published. Please let me know if it works alright and if you're still experiencing, feel free to open an issue over at the [minimap-codeglance](https://github.com/olmokramer/atom-minimap-codeglance) repo.

**Comment by abe33** [comment]
@olmokramer Thanks for the quick fix, I can confirm that it fixed both cases, I'll close this issue then.
